typed-rest-client/HttpClient missing from ContainerStuctureTest task

*Type*: Bug

**Enter Task Name**: ContainerStructureTestV0  

## Environment
- Azure Pipelines
- Agent - Hosted 

```    
pool:
  name: Hosted Ubuntu 1604
```

## Issue Description

Task fails:

```
Starting: Container Structure Test
==============================================================================
Task         : Container Structure Test
Description  : Uses container-structure-test (https://github.com/GoogleContainerTools/container-structure-test) to validate the structure of an image based on four categories of tests - command tests, file existence tests, file content tests and metadata tests
Version      : 0.179.0
Author       : Microsoft Corporation
Help         : https://aka.ms/containerstructuretest
==============================================================================
##[error]Unhandled: Cannot find module 'typed-rest-client/HttpClient'
Finishing: Container Structure Test
```

## Workaround
https://github.com/microsoft/azure-pipelines-tasks/issues/14247#issuecomment-768834565

--- a/Tasks/ContainerStructureTestV0/package.json
+++ b/Tasks/ContainerStructureTestV0/package.json
@@ -21,6 +21,7 @@
     "@types/q": "^1.0.7",
     "azure-pipelines-task-lib": "2.8.0",
     "azure-pipelines-tasks-docker-common-v2": "1.0.0",
+    "typed-rest-client": "1.5.0",
     "utility-common": "file:../../_build/Tasks/Common/utility-common-1.0.2.tgz",
     "utility-common-v2": "file:../../_build/Tasks/Common/utility-common-v2-2.0.0.tgz",
     "uuid": "^3.0.1"

--- a/Tasks/ContainerStructureTestV0/testrunner.ts
+++ b/Tasks/ContainerStructureTestV0/testrunner.ts
@@ -69,7 +69,7 @@ export class TestRunner {
     }
 
     private runContainerStructureTest(runnerPath: string, testFilePath: string, image: string): string {
-        var tool:tr.ToolRunner = tl.tool(runnerPath).arg(["test", "--image", image, "--config", testFilePath, "--json"]);
+        var tool:tr.ToolRunner = tl.tool(runnerPath).arg(["test", "--image", image, "--config", testFilePath, "--output", "json"]);
         let output = undefined;
 
         try {

